install-guides/ubuntu20.md

```diff
@@ -0,0 +1,512 @@
+```
+![image](https://user-images.githubusercontent.com/17537000/171808166-400b5299-1c90-43af-aa14-5daa76398ab0.png)
+
+
+## Part 5.2 - PHP-FPM - Setup
+* Make a copy of the php-fpm pool config file for pixelfed
+```
+cp /etc/php/7.4/fpm/pool.d/www.conf /etc/php/7.4/fpm/pool.d/pixelfed.conf
+```
+
+* Open the php-fpm pixelfed.conf file
+```
+nano /etc/php/7.4/fpm/pool.d/pixelfed.conf
+```
+
+* Edit these lines
+```
+[pixelfed]
+user = pixelfed
+group = pixelfed
+listen = /run/php/php7.4-fpm-pixelfed.sock
+```
+![image](https://user-images.githubusercontent.com/17537000/171807724-84a5d028-a6d2-48f9-9a83-19a4345406fb.png)
+
+* Restart php-fpm
+```
+systemctl restart php7.4-fpm
+```
+```
+systemctl status php7.4-fpm
+```
+![image](https://user-images.githubusercontent.com/17537000/171828436-a5c41ae9-33e1-4973-84e2-e2c1d5c604ee.png)
+
+## Part 6 - Install Composer
+* Download the composer installer to /tmp/
+```
+curl -sS https://getcomposer.org/installer -o /tmp/composer-setup.php
+```
+
+* Install composer
+```
+php /tmp/composer-setup.php --install-dir=/usr/local/bin --filename=composer
+```
+![image](https://user-images.githubusercontent.com/17537000/171808535-b0a08f87-2b73-436f-91e7-133e7074a16d.png)
+
+
+## Part 8 - Installation of Pixelfed
+* Login as the new pixelfed user
+```
+su - pixelfed
+```
+![image](https://user-images.githubusercontent.com/17537000/171809094-7d5b7b1d-13ff-4eee-bebd-b496c5590d9b.png)
+
+## Part 7.1 - Prepare your new Pixelfed instance as the pixelfed user
+* Using git, clone a copy of the repo from the dev branch into the home folder of the new pixelfed user.
+```
+git clone -b dev https://github.com/pixelfed/pixelfed.git pixelfed && cd pixelfed
+```
+![image](https://user-images.githubusercontent.com/17537000/171809397-9c797416-1b33-4069-b5a6-e63569006802.png)
+
+* Install all of the php dependant packages using composer
+```
+composer install --no-ansi --no-interaction --optimize-autoloader
+```
+![image](https://user-images.githubusercontent.com/17537000/171809700-53d77822-bb8f-475d-812e-3e7f6c5ae086.png)
+
+## Part 8 - Prepare new Pixelfed instance
+* Copy the default .env file
+```
+cp /home/pixelfed/pixelfed/.env.example /home/pixelfed/pixelfed/.env
+```
+
+* Open the .env file
+```
+nano /home/pixelfed/pixelfed/.env
+```
+
+
+* Edit these lines to match your new instance
+```
+APP_NAME="Pixelfed Australia"
+APP_DEBUG=false
+
+APP_URL="https://pixelfed.au"
+APP_DOMAIN="pixelfed.au"
+ADMIN_DOMAIN="pixelfed.au"
+SESSION_DOMAIN="pixelfed.au"
+          
+DB_PASSWORD=secretpasswordhere
+```
+
+## Part 8.1 - Unix Socket (Redis and MariaDB)
+* Coming soon
+<!-- * Edit these lines to match your new instance
+```
+    REDIS_SCHEME=unix
+    REDIS_PATH=/var/run/redis/redis-server.sock
+    REDIS_HOST=null
+    REDIS_PASSWORD=null
+    REDIS_PORT=null
+```
+![image](https://user-images.githubusercontent.com/17537000/171834219-cb27183d-374d-4c61-b987-c09bfa29b797.png) -->
```

Comment:
don't forget to uncomment when you're done

Reply:
Done